### PR TITLE
Port System.Runtime.CompilerServices & InteropServices RC1 docs

### PR DESCRIPTION
**Pull request by carlossanlop**
@dotnet/interop-contrib @dotnet/area-system-runtime-compilerservices PTAL.
Please review ignoring whitespace.

### CODE REVIEW

**Review by jkoritzinsky — APPROVED** [review]
InteropServices LGTM

File: xml/System.Runtime.CompilerServices/IsUnmanagedAttribute.xml [hunk 1]
```diff
@@ -1,49 +1,50 @@
...
+      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; IsUnmanagedAttribute();" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime</AssemblyName>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters />
+      <Docs>
+        <summary>Initializes the attribute.</summary>
```
Comment (gewarren) [inline xml/System.Runtime.CompilerServices/IsUnmanagedAttribute.xml:45]:
```suggestion
        <summary>Initializes a new instance of <see cref="T:System.Runtime.CompilerServices.IsUnmanagedAttribute" />.</summary>
```

File: xml/System.Runtime.CompilerServices/NullableAttribute.xml [hunk 2]
```diff
@@ -1,102 +1,102 @@
...
+      <MemberType>Field</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime</AssemblyName>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Byte[]</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Flags specifying metadata related to nullable reference types.</summary>
```
Comment (gewarren) [inline xml/System.Runtime.CompilerServices/NullableAttribute.xml:97]:
```suggestion
        <summary>Specifies metadata related to nullable reference types.</summary>

```

File: xml/System.Runtime.CompilerServices/NullableContextAttribute.xml [hunk 3]
```diff
@@ -1,73 +1,73 @@
...
+      <MemberType>Field</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime</AssemblyName>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Byte</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Flag specifying metadata related to nullable reference types.</summary>
```
Comment (gewarren) [inline xml/System.Runtime.CompilerServices/NullableContextAttribute.xml:68]:
```suggestion
        <summary>Specifies metadata related to nullable reference types.</summary>

```

File: xml/System.Runtime.InteropServices.Marshalling/StrategyBasedComWrappers.xml [hunk 4]
```diff
@@ -1,279 +1,280 @@
...
+            </Attribute>
+          </Attributes>
+        </Parameter>
+        <Parameter Name="flags" Type="System.Runtime.InteropServices.CreateComInterfaceFlags" />
+        <Parameter Name="count" Type="System.Int32" RefType="out" />
+      </Parameters>
+      <Docs>
+        <param name="obj">Target of the returned Vtables.</param>
+        <param name="flags">Flags used to compute Vtables.</param>
+        <param name="count">The number of elements contained in the returned memory.</param>
```
Comment (gewarren) [inline xml/System.Runtime.InteropServices.Marshalling/StrategyBasedComWrappers.xml:87]:
```suggestion
        <param name="count">When this method returns, contains the number of elements contained in the returned memory.</param>
```

File: xml/System.Runtime.InteropServices.Marshalling/StrategyBasedComWrappers.xml [hunk 5]
```diff
@@ -1,279 +1,280 @@
...
+              <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.Nullable(1)&gt;]</AttributeName>
+            </Attribute>
+          </Attributes>
+        </Parameter>
+        <Parameter Name="flags" Type="System.Runtime.InteropServices.CreateComInterfaceFlags" />
+        <Parameter Name="count" Type="System.Int32" RefType="out" />
+      </Parameters>
+      <Docs>
+        <param name="obj">Target of the returned Vtables.</param>
+        <param name="flags">Flags used to compute Vtables.</param>
```
Comment (gewarren) [inline xml/System.Runtime.InteropServices.Marshalling/StrategyBasedComWrappers.xml:86]:
```suggestion
        <param name="flags">A bitwise combination of the enumeration values that specifies how to compute Vtables.</param>
```

File: xml/System.Runtime.InteropServices.Marshalling/StrategyBasedComWrappers.xml [hunk 6]
```diff
@@ -1,279 +1,280 @@
...
+      <ReturnValue>
+        <ReturnType>System.Object</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="externalComObject" Type="System.IntPtr" />
+        <Parameter Name="flags" Type="System.Runtime.InteropServices.CreateObjectFlags" />
+      </Parameters>
+      <Docs>
+        <param name="externalComObject">Object to import for usage into the .NET runtime.</param>
+        <param name="flags">Flags used to describe the external object.</param>
```
Comment (gewarren) [inline xml/System.Runtime.InteropServices.Marshalling/StrategyBasedComWrappers.xml:160]:
```suggestion
        <param name="flags">A bitwise combination of the enumeration values that describes the external object.</param>
```

**Review by gewarren — COMMENTED** [review]
I want to wait to check the rendered example before approving.

**Review by gewarren — APPROVED** [review]

**Pull request merged**